src/utils/getApiStaticPath.ts

```diff
@@ -1,4 +1,4 @@
-import { API_CATEGORIES, API_SUB_CATEGORIES } from '@/data/api-categories';
+import { API_CATEGORIES, API_SUB_CATEGORIES } from '@/data/api-categories.mjs';
```

Comment:
Is this necessary?

Reply:
Interesting. If I remove the file type,  then I get an error

[src/utils/getApiStaticPath.ts](https://github.com/aws-amplify/docs/pull/8229/files/55315e08aeae237546425e17b1ef7bdf1b1fe29d#diff-633412ab065a95d2960c2b1c55d2e0d97a477889415fe780f3d1fd8f76925e52)

There wasn't a `*.d.ts` file there before so I'm not sure why the prior version worked :-/